dev-tools/src/main/resources/ant/integration-tests.xml

```diff
@@ -87,15 +88,20 @@
       <run-script script="@{home}/bin/plugin">
         <nested>
           <arg value="install"/>
+          <arg value="-Des.path.plugins=${path.plugins}" if:set="path.plugins"/>
+          <arg value="-Des.path.conf=${path.conf}" if:set="path.conf"/>
+          <arg value="-Des.path.logs=${path.logs}" if:set="path.logs"/>
           <arg value="${url}"/>
         </nested>
       </run-script>
+      <property if:set="path.plugins" name="plugins.installfolder" value="${path.plugins}"/>
```

Comment:
I think you can use `<condition>` instead of antcontrib if/unless? Something like:
```
<condition property="path.installfolder" value="${path.plugins}" else="@{home}/plugins">
  <isset property="path.plugins"
</condition>
```

But should this be a local then?